tests/lib/snaps/basic-iface-hooks-producer/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: basic-iface-hooks-producer
+version: 1.0
+slots:
+    bar:
+        interface: content
+        content: y
+        read:
+            - /etc
```

Comment:
In addition, can you please use `$SNAP/etc` and `$SNAP/usr` (this is what your declaration meant anyway).